Goal: Fix documentation links

Fixes: #508388

Status wrt upstream: Debian-specific

Note: 

Index: samba-3.2.5/docs/htmldocs/index.html
===================================================================
--- samba-3.2.5.orig/docs/htmldocs/index.html
+++ samba-3.2.5/docs/htmldocs/index.html
@@ -30,8 +30,4 @@
  <td valign="top"><a href="manpages/index.html">Man pages</a></td>
  <td valign="top">The Samba man pages in HTML.</td>
 </tr>
-<tr>
- <td valign="top"><a href="../../WHATSNEW.txt">WHATSNEW</a></td>
-  <td valign="top">Samba Release Notes.</td>
-</tr>
 </table></body></html>
